src/triage/component/results_schema/README.md

```diff
@@ -7,31 +7,35 @@ Store results of modeling runs in a relational database
 
 `pip install git+https://github.com/dssg/triage.git`
 
-2. Create a YAML file with your database credentials (see example_db_config.yaml), or an environment variable 'DBURL' with a connection string. The database must be created already.
+2. You can do the initial schema and table creation a couple of different ways, with a couple of different options for passing database credentials.
 
-3. Call 'upgrade_db' function from Python console or script
+	- *Triage cli script*. Assuming you have triage installed from pip: `triage upgrade_db -d dbcredentials.yaml`, or just `triage upgrade_db` if you have a database credentials file at `database.yaml` (the default)
+	- *From Python code*. If you are in a Python console or notebook, you can call `upgrade_db` with either sqlalchemy engine pointing to your database, or a filename similar to what's used to launch the cli script.
 
-```
->>> from triage.component.results_schema import upgrade_db
->>> upgrade_db('my_db_config.yaml')
-```
+	```
+	>>> from triage.component.results_schema import upgrade_db
+	>>> upgrade_db(engine)
+	>>> upgrade_db('database.yaml')
+	```
 
 This command will create a 'results' schema and the necessary tables.
 
 
 ## Modifying the schema
 
-[Alembic](http://alembic.zzzcomputing.com/en/latest/tutorial.html) is a schema migrations library written in Python. It allows us to auto-generate migrations to run incremental database schema changes, such as adding or removing a column. This is done by comparing the definition of a schema in code with that of a live database. There are many valid ways to create migrations, which you can read about in [Alembic's documentation](http://alembic.zzzcomputing.com/en/latest/tutorial.html). But here is a common workflow we will use to modify the schema.
+To make modifications to the schema, you should be working in a cloned version of the repository.
```

Comment:
Sorry I think I'm a little confused about this. Is it not expected to be fairly typical for people to want to modify the schema? And aren't they working in _their own_ project repositories, with Triage just a dependency?

Or, is this really just about Triage development?

Anyway, it seems like we can provide this through the installed CLI.

(But, I'm not a user, and I really don't know.)

Reply:
Triage development. There isn't really a use case for a triage user (e.g. a project running triage on their data) to modify the results schema.

Reply:
:+1: awesome :+1: